src/Helpers.ts

```diff
@@ -0,0 +1,25 @@
+/** function latinCheck*/
+// no cyrillic letters allowed
+function latinCheck(string: string): boolean {
+    const cyrillic = /.*[а-яА-ЯёЁ]/g;
+
+    return string.search(cyrillic) === -1;
+  }
+
+  /** function emailCheck*/
+  // email must contain latin letters, numbers, dot, gap and dash. domain has at least 2 letters.
+  // parts of email are separated with @ sign
+  export function emailCheck(email: string): boolean {
```

Comment:
Или, если функция что-то проверяет, можно называть в формате `is<What>`?
```suggestion
  export function isEmail(email: string): boolean {
```